guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,173 @@
+[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+
+Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
+LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
+* {Project} installations running on {RHEL} 7 cannot upgrade to {RHEL} 8.
+You must have a {RHEL} compatible LEAPP build to upgrade.
+* Access to available repositories or a local mirror of repositories.
+* No outstanding updates.
+* {Project} installations can be upgraded to CentOS Stream 8 and a {RHEL} rebuild.
+
+.Procedure
+[start=1]
+. Configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer LEAPP packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} upgrades:
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
+[start=2]
+. Install required packages:
++
+----
+# yum install leapp leapp-repository leapp-data-centos
+----
+
+[start=3]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+. Add {Project} specific repositories to `/etc/leapp/files/leapp_upgrade_repositories.repo`:
++
+----
+[leapp-foreman]
+name=Foreman 3.2
+baseurl=https://yum.theforeman.org/releases/3.2/el8/$basearch
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+
+ifdef::katello[]
+[leapp-katello]
+name=Katello 4.4
+baseurl=https://yum.theforeman.org/katello/4.4/katello/el8/$basearch/
```

Comment:
This document will work for Katello 4.4 and 4.5, so I think we should use `{KatelloVersion}` here.